hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/server/JournalMetrics.java

```diff
@@ -52,11 +52,7 @@
   MutableCounterLong bytesServedViaRpc;
 
   @Metric
-  MutableStat rpcRequestCacheMissAmount = new MutableStat(
-      "RpcRequestCacheMissAmount", "Number of RPC requests unable to be " +
-      "served due to lack of availability in cache, and how many " +
-      "transactions away the request was from being in the cache.",
-      "Misses", "Txns");
+  MutableStat rpcRequestCacheMissAmount;
```

Comment:
If we explicitly instantiate the metric via `registry.newStat()`, then we can remove the `@Metric` annotation. It's only necessary for metrics2 to automatically create the metric for us. (It's been a while since I've looked at Hadoop or metrics2, so let me know if I'm mis-remembering here)

Reply:
Thanks for review! You are right, I will remove it later.